ci-operator/step-registry/ipi/conf/capability/baselinecaps/ipi-conf-capability-baselinecaps-commands.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+openshift-install version
+declare -a baseline_caps_set_array
+IFS="," read -r -a baseline_caps_set_array <<< "$(openshift-install explain installconfig.capabilities.baselineCapabilitySet | grep 'Valid Values' | awk -F':' '{print $2}' | tr -d '"')"
+
+if [[ ${#baseline_caps_set_array[@]} -eq 0 ]]; then
+    echo "Unable to get valid values for baselineCapabilitySet!"
+    exit 1
+fi
+
+#Remove None from array
```

Comment:
So this step is only used to cover v4.x baseline capabilities in a random way, right? If that, shall we remove the "default" one together with "None"?

Reply:
If we want to use this step only to cover `v4.x` baseline capabilities, suggest to indicate it in the step name, e.g: `ipi-conf-capability-baselinecaps-v4xx`

Reply:
>  shall we remove the "default" one together with "None"?

Here "default" one means "vCurrent"? we have case "vCurrent" + "addtionalCaps", so only remove None.
OCP-49193 [installer] install cluster with baselineCapabilitySet as vCurrent and set some caps in additionalEnabledCapabilities

Reply:
> Here "default" one means "vCurrent"?

Right

> we have case "vCurrent" + "addtionalCaps"

Got it, but I think we still can rename it step name to indicate what is trying to cover.

Reply:
sure, updated to `ipi-conf-capability-baselinecaps-v-capset`